dm_formatter.py

```diff
@@ -0,0 +1,84 @@
+# Convert drexel_metadata JSON file to BGNN minnow metadata JSON file
+import json
+import argparse
+
+
+def reformat_for_bgnn(result):
+    """
+    Reformat and reduce the size of the result dictionary. 
+    Collect only the data necessary for BGNN minnow project. The new format matches the 
+    BGNN_metadata version. Therefore some of the value not calcualted in drexel version are by 
+    defaulset to "None". 
+
+    Parameters
+    ----------
+    result : dict
+        DESCRIPTION. output from gen_metadata()
+
+    Returns
+    -------
+    bgnn_result : dict
+        DESCRIPTION. {'base_name': xx, 'version':xx, 
+                       'fish': {'fish_num': xx,"bbox":xx, 'pixel_analysis':xx, 'rescale':xx, 
+                            'eye_bbox': xx, 'eye_center':xx , 'angle_degree': xx,
+                            'eye_direction':xx, 'foreground_mean':xx, 'background_mean':xx}, 
+                       'ruler': {'bbox':xx, 'scale':xx, 'unit':xx}}
+
+    """
+    
+    name_base = list(result.keys())[0]
+    first_value = list(result.values())[0]
+    
+    # Fish metadata
+    fish_num = first_value['fish_count']
+    fish_bbox = first_value['fish'][0]['bbox']
+    pixel_analysis = False if first_value['fish'][0]['pixel_analysis_failed'] else True
```

Comment:
Wouldn't it be simpler to just use this assignment?
```suggestion
    pixel_analysis = !first_value['fish'][0]['pixel_analysis_failed']
```

Reply:
FYI: This should have been `pixel_analysis = not first_value['fish'][0]['pixel_analysis_failed']`.